tools-local/ILAsmVersion.txt

```diff
@@ -0,0 +1 @@
+2.1.0-preview2-26314-17
```

Comment:
I think we should set up the updater for this: https://github.com/dotnet/corefx/blob/305bebcb8e9c4c0b05ecf6b2f87fa723f1a33493/dependencies.props#L183-L187

Reply:
good call I will do that.